libs/js/Date.jsx

```diff
@@ -0,0 +1,53 @@
+import React, {Component} from 'react';
+import {
+  FormGroup,
+  FormControl,
+  HelpBlock
+} from 'react-bootstrap';
+export default class Text extends Component {
```

Comment:
Can you replace class Text with Date?

Reply:
Alright, good catch. Will take care of this.